readme.md

```diff
@@ -166,15 +166,17 @@ Explicit 256/Truecolor mode can be enabled using the `--color=256` and `--color=
 
 ### Modifiers
 
-- `reset`
-- `bold`
-- `dim`
-- `italic` *(Not widely supported)*
-- `underline`
-- `inverse`
-- `hidden`
-- `strikethrough` *(Not widely supported)*
-- `visible` (Text is emitted only if enabled)
+| Modifier | Definition |
+| --- | --- |
+| `reset` | Resets the current color chain. |
+| `bold` | Make text bold. |
+| `dim` | Emitting only a small amount of light. |
+| `italic` | Make text italic. *(Not widely supported)* |
+| `underline` | Make text underline. *(Not widely supported)* |
+| `inverse` | Inverse background and foreground colors. |
+| `hidden` | Prints the text, but makes it invisible. |
+| `strikethrough` | [https://en.wikipedia.org/wiki/Strikethrough](https://en.wikipedia.org/wiki/Strikethrough) |
+| `visible` | Prints the text only when Chalk is enabled. Can be useful for things that are purely cosmetic. |
```

Comment:
Can we not use tables here? They're annoying to maintain and doesn't give much benefit.

You can just use the format:

- `reset` - Resets the current color chain.